Src/WitsmlExplorer.Frontend/components/ContentViews/JobsView.tsx

```diff
@@ -131,11 +144,12 @@ export const JobsView = (): React.ReactElement => {
             ),
             targetServer: serverUrlToName(servers, jobInfo.targetServer),
             sourceServer: serverUrlToName(servers, jobInfo.sourceServer),
-            report: jobInfo.report ? (
-              <ReportButton onClick={() => onClickReport(jobInfo.report)}>
-                Report
-              </ReportButton>
-            ) : null,
+            report:
+              jobInfo.status === "Finished" ? (
+                <ReportButton onClick={() => onClickReport(jobInfo.id)}>
+                  {jobInfo.linkType}
```

Comment:
Add a corresponding enum in the frontend, and decide the text based on that enum.